.travis.yml

```diff
@@ -40,20 +44,44 @@ before_install:
 script:
   - |
     bazel \
-      --output_base=$HOME/.cache/bazel \
+      --output_base=$HOME/.bazel-output-base \
       --batch \
       --host_jvm_args=-Xmx500m \
       --host_jvm_args=-Xms500m \
-      test \
+      build \
       tensorboard/... \
       --worker_verbose \
       --verbose_failures \
-      --test_output=errors \
       --spawn_strategy=sandboxed \
       --genrule_strategy=sandboxed \
       --local_resources=400,2,1.0 \
+      --worker_max_instances=2 \
       --strategy=Javac=worker \
       --strategy=Closure=worker
+  - |
+    bazel \
+      --output_base=$HOME/.bazel-output-base \
+      --batch \
+      --host_jvm_args=-Xmx500m \
+      --host_jvm_args=-Xms500m \
+      test \
+      tensorboard/... \
+      --verbose_failures \
+      --test_output=errors \
+      --spawn_strategy=sandboxed \
+      --local_resources=400,2,1.0
+
+before_cache:
+  - |
+    find $HOME/.bazel-output-base \
+      -name \*.runfiles \
+      -or -name \*.tar.gz \
+      -or -name \*-execroot.json \
+      -or -name \*-tsc.json \
+      -or -name \*-params.pbtxt \
+      -or -name \*-args.txt \
+      -or -name \*.runfiles_manifest \
+      | xargs rm -rf
```

Comment:
Unsafe. Use `find [args] -print0 | xargs -0 rm -rf`, or, in my opinion better yet, just `find [args] -exec rm -rf {} +`. (Yes, like `xargs`, `find -exec +` will make sure to batch commands so as not to exceed `ARG_MAX`.)